utilslib/dr.py

```diff
@@ -206,8 +207,11 @@ def __init__(self, *args, **kwargs):
         K8s object
         """
         super(K8s, self).__init__(*args, **kwargs)
+        
         # Use in cluster config when deployed to cluster
         config.load_kube_config()
+        cfg = config.kube_config.list_kube_config_contexts()
+        self.cluster_name = cfg[0][0]['context']['cluster'].split("/")[1]
```

Comment:
I'm guessing the above line gets all the contexts in the kubeconfig and this assumes the current context is the first in the list of contexts. Which it may not be. But we could cross that bridge if it is ever and issue.

Reply:
As discussed, will replace this when in cluster